net/respondd/src/respondd.c

```diff
@@ -419,6 +584,14 @@ int main(int argc, char **argv) {
 			join_mcast(sock, mgroup_addr, optarg);
 			break;
 
+		case 't':
+			max_multicast_delay = 1000 * atoi(optarg);
```

Comment:
Better use `strtoul()`. That gives you the positivity check for free and lets you catch strings that are not numbers and thus invalid. (Yes, the same would apply for `-p` above)